workspaces/github-actions/plugins/github-actions/src/alpha/entityCards.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { createSchemaFromZod } from '@backstage/frontend-plugin-api';
+import { createEntityCardExtension } from '@backstage/plugin-catalog-react/alpha';
+
+/**
+ * @alpha
+ */
+export const entityGithubActionsCard = createEntityCardExtension({
+  name: 'workflow-runs',
+  loader: () =>
+    import('../components/Router').then(m => <m.Router view="cards" />),
+});
+
+/**
+ * @alpha
+ */
+export const entityLatestGithubActionRunCard = createEntityCardExtension({
+  name: 'latest-workflow-run',
+  configSchema: createSchemaFromZod(z =>
+    z.object({
+      props: z
+        .object({
+          branch: z.string().default('master'),
+          variant: z.enum(['flex', 'fullHeight', 'gridItem']).optional(),
+        })
+        .default({}),
+      filter: z.string().default(''),
```

Comment:
Afaik this should be `filter: z.string().optional()` here and in other places, does that work too?